Interfaces/API/Dirac.py

```diff
@@ -803,7 +745,7 @@ def runLocal(self, job):
         return self._errorReport( 'Could not retrieve DIRAC/VOPolicy/InputDataModule for VO' )
 
       self.log.info( 'Job has input data requirement, will attempt to resolve data for %s' % DIRAC.siteName() )
-      self.log.verbose( '\n'.join( inputData ) )
+      self.log.verbose( '\n'.join( inputData if isinstance(inputData, (list, tuple)) else [inputData] ) )
```

Comment:
should `dict` be part of the `isinstance` ?

Reply:
I don't think so.